utilities/virt.py

```diff
@@ -1063,6 +1049,13 @@ def wait_for_specific_status(self, status, timeout=TIMEOUT_3MIN, sleep=TIMEOUT_5
     def privileged_vmi(self):
         return VirtualMachineInstance(client=get_client(), name=self.name, namespace=self.namespace)
 
+    def wait_for_agent_connected(self, timeout: int = TIMEOUT_5MIN):
```

Comment:
In wait_for_guest_agent the timeout was TIMEOUT_12MIN. Are you sure 5 minutes is enough?

Reply:
That default was never used because the only usage was from `wait_for_vm_interface` which explicitly called it with the same insane timeout of 12min.

The 5min is the default seen on the wrapper.

Reply:
IIRC, the 12 minutes was originally introduced as a W/A for some infrastructure slowness, but I agree it's not an acceptable timeout to wait for a VM to come up.
It will be interesting, however, to see is this new timeout introduces new failures.

Reply:
It will show up on new tests, not old/existing ones. So there is no regression in this regard.

Reply:
The function was internal and never used from outside, so I do not understand this long thread about this topic.

Do you have a specific argument why we need a 12min default timeout for waiting on a VM guest to boot?

Reply:
since the original timeout is not changed inside `wait_for_vm_interfaces` function - I'm ok to have 5 min here. 

12 min timeout was implemented so long ago.. Mostly for slow infra and Windows on PSI clusters, and I'm pretty sure the real reason for it may not even exist anymore, but since `wait_for_vm_interfaces` is used in many places - it would be better to keep original timeout there.

for `wait_for_agent_connected` - 5min sounds ok.